source/agora/common/crypto/ECC.d

```diff
@@ -90,11 +91,46 @@ public struct Scalar
         crypto_core_ed25519_scalar_reduce(this.data[].ptr, param[].ptr);
     }
 
-    /// Expose `toString`
-    public void toString (scope void delegate(const(char)[]) @safe sink)
-        const @safe
+    /// Describe print modes for `toString`
+    public enum PrintMode
+    {
+        /// Default value, print hashed version
```

Comment:
```suggestion
        /// Print shortened hashed version
```

The default value is in the parameter itself. And users know that the first member of an enum is the enum's `.init` value, it doesn't need to be explained here.